torch/_dynamo/variables/tensor.py

```diff
@@ -217,15 +217,25 @@ def try_generic_attr_handling():
 
         return result
 
+    def has_unpack_var_sequence(self, tx):
+        return (self.size is not None and len(self.size) > 0) or (
```

Comment:
Can't you technically unpack the sequence even if it has no elements?

Reply:
Yeah, scalar is a different thing; your code will error here too because size(0) will fail

Reply:
I think I'm doing it right by checking `len(size)`, no?
* scalar case: `len(size) == 0`; don't try to unpack it
* empty sequence: `len(size) == 1` with `size(0) == 0`; okay to unpack

Reply:
Oh oopsies, you are right!